apps/postybirb-ui/.env.development

```diff
@@ -0,0 +1,2 @@
+# On development, we load page from vite http server
+VITE_CSP="default-src http://localhost:* ws:*; style-src *"
```

Comment:
Note: I tried to implement CSP to remove that annoying log message but it was not working 😞